build/build.go

```diff
@@ -90,7 +90,10 @@ func Import(path string, mode build.ImportMode, installSuffix string, buildTags
 	return pkg, nil
 }
 
-func Parse(pkg *build.Package, fileSet *token.FileSet) ([]*ast.File, error) {
+// parse parses and returns all .go files for given pkg, including all transitive dependencies.
```

Comment:
After looking at the code more, I think that's incorrect.

I originally thought `Parse` was including all dependencies and all dependencies of dependencies, recursively. But now I see it's only including the files of the given package. However, it does special processing for standard library packages that have some code in `natives` directory.

So yeah, that comment is wrong and should be changed. This is why describing what this func does will helpful, since it takes time and effort to figure it out from code.